client/ayon_houdini/plugins/publish/extract_rop.py

```diff
@@ -78,13 +78,32 @@ def update_representation_data(self,
         pass
 
 
-class ExtractOpenGL(ExtractROP,
-                    publish.ColormanagedPyblishPluginMixin):
+class ExtractOpenGLAndFlipbook(ExtractROP,
+                               publish.ColormanagedPyblishPluginMixin):
 
     order = pyblish.api.ExtractorOrder - 0.01
-    label = "Extract OpenGL"
+    label = "Extract Review (OpenGL & Flipbook)"
     families = ["rop.opengl"]
```

Comment:
I'm not too sure about both these node types going through `rop.opengl` family - they should each get their own family so that we could potentially still target just one of the two if we get to any specific validations down the line. Not critical for this PR - but worth a low prio backlog issue.

Reply:
Added a todo here https://github.com/ynput/ayon-houdini/pull/167/commits/979452c4dee0a1769be2ed418b1897467ed10524
and also here's an issue https://github.com/ynput/ayon-houdini/issues/176